test/unit/expressions/expressionCombinedTransformations-test.js

```diff
@@ -192,7 +192,7 @@ describe('Combine Jexl and legacy expressions (default JEXL)', function () {
                 .matchHeader('fiware-servicepath', 'gardens')
                 .post(
                     '/v1/updateContext',
-                    utils.readExampleFile('./test/unit/examples/contextRequests/updateContextExpressionPlugin3.json')
+                    utils.readExampleFile('./test/unit/examples/contextRequests/updateContextExpressionPlugin3b.json')
```

Comment:
Instead of creating a new file (with the b suffix) I think it would be getter to edit the current ones. This way, the removal done in PR https://github.com/telefonicaid/iotagent-node-lib/pull/995 would be easy (as there files are currently taken into account for the removal).

Reply:
IMHO is not possible edit current file, and should be splited to use in different tests. 
Renamed files done into 8f3ea370c13ebee1ec8c5b777523168af70f65f0

Reply:
Ok. I see (for instance https://github.com/telefonicaid/iotagent-node-lib/search?q=updateContextExpressionPlugin3).

I'll comment about the new files added in this PR on #995

Reply:
> I'll comment about the new files added in this PR on #995

Here: https://github.com/telefonicaid/iotagent-node-lib/pull/995#issuecomment-840368989

NTC